### PR TITLE
Fixes an improper markup

### PR DESCRIPTION
**Pull request by hiwakaba**
Hello, this PR fixes  an improper markup.

Signed-off-by: Hirotaka Wakabayashi <hiwkby@yahoo.com>

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@hiwakaba Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=1116)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=1116) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@hiwakaba Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=1116)!

**Review by michaelklishin — APPROVED** [review]
Ahem, I'd rephrase the whole thing but OK.
